package.json

```diff
@@ -50,6 +51,7 @@
     "ethereum-waffle": "^4.0.10",
     "ethers": "^6.11.1",
     "hardhat": "2.22.1",
+    "hardhat-contract-sizer": "^2.10.0",
```

Comment:
remove if not needed, don't want to depend on random packages if we can avoid it

Reply:
this package can be used to get the contract size. It was helpful initially when the contracts were quite bulky over 24kb size.